File: API/Entities/SurveyForm.cs
namespace API.Entities
{
    public class SurveyForm
    {
        public int Id { get; set; }
        public string Title { get; set; } = string.Empty;
        public string Path { get; set; } = string.Empty;
        public string ImagePath { get; set; } = string.Empty;
        public DateTime Created { get; set; } = DateTime.UtcNow;
        public ICollection<SurveyFormDetail> SurveyFormDetails { get; set; } = [];
    }
}

File: client/src/app/_models/survey-form.ts
import { SurveyFormDetail } from "./survey-form-detail";

export interface SurveyForm {
    id: number
    title: string;
    path: string;
    clientPath: string;
    imagePath: string;
    created: Date;
    surveyFormDetails: SurveyFormDetail[];
}
